app/src/providers/property/mapMarkerDataParser.ts

```diff
@@ -0,0 +1,24 @@
+import _ from "lodash";
+import { Property } from "api/codegen";
+
+import { PropertyMapMarker } from "app/map/Map.types";
+
+export function mapMarkerPropertiesDataParser(properties: Array<Property>): Array<PropertyMapMarker> | [] {
+  if (!properties?.length) return [];
+
+  const parsedPropertiesInfo = properties?.map((property) => {
```

Comment:
If we are asserting for empty properties above, you can do this instead:

```suggestion
  const parsedPropertiesInfo = properties!.map((property) => {
```